src/encoded/commands/clear_db_es_contents.py

```diff
@@ -37,9 +38,10 @@ def clear_db_tables(app):
     meta.reflect()
     connection = session.connection().connect()
     try:
-        # truncate tables by only deleting contents
-        for table in meta.sorted_tables:
-            connection.execute(table.delete())
+        # truncate tables by only deleting contents (sqlalchemy 1.4+ compliant)
+        table_names = ','.join(table.name for table in reversed(Base.metadata.sorted_tables))
+        connection.execute('SET statement_timeout = 300000;')  # give 5 mins for DB clear
+        connection.execute(f'TRUNCATE {table_names} RESTART IDENTITY;')
```

Comment:
1. Is there any need for a CASCADE argument here?
2. should you be doing this as a single transaction so that it doesn't try to do conflict detection until you're done?
Maybe neither of these matter but I wanted to ask the question. I guess if you're getting OK results it probably doesn't matter. 

(Personally, I'm inclined to think you should DROP the DB and copy in a fresh empty DB from elsewhere rather than even doing these actions. I think actions like these are all risks of pointless inconsistency detection or even of loss of time proving that won't happen. I think copying a new DB into place of an old one is likely to be _way_ faster than this. It would be OK to just insert a comment suggesting this be explored in the future.)